src/Sentry/Laravel/Features/QueueIntegration.php

```diff
@@ -180,22 +185,19 @@ public function handleJobProcessingQueueEvent(JobProcessing $event): void
 
         $resolvedJobName = $event->job->resolveName();
 
-        $receiveLatency = null;
-        if ($event->job->payload()[self::QUEUE_PAYLOAD_PUBLISH_TIME] !== null) {
-            $receiveLatency = microtime(true) - $event->job->payload()[self::QUEUE_PAYLOAD_PUBLISH_TIME];
-        }
+        $jobPublishedAt = $jobPayload[self::QUEUE_PAYLOAD_PUBLISH_TIME] ?? null;
 
         $job = [
             'messaging.system' => 'laravel',
-            
+
             'messaging.destination.name' => $event->job->getQueue(),
             'messaging.destination.connection' => $event->connectionName,
-            
-            'messaging.message.id' => (string) $event->job->getJobId(),
+
+            'messaging.message.id' => $jobPayload['uuid'] ?? $event->job->getJobId(),
```

Comment:
We need to keep the string cast for `$event->job->getJobId()`.

Reply:
It's documented as returning a string? When doesn't it do that?